### ISSUE TITLE
Error message about hand-entered authority URI preventing conversion

### ISSUE THREAD

**Issue by NancyL** [LD4P/rdf2marc#177]
**Type of change**
Defect?

**Example RDF records**
Please provide URIs or RDF of the `bf:Instance`. If providing RDF, Turtle is preferred.

https://api.stage.sinopia.io/resource/ec2057de-dbed-4407-af3a-ad71445e380e

**Resource template**
Which resource template does this apply to? In which Sinopia environment (development, stage, production)?

Stanford:bf2:Monograph:Instance/Stanford:bf2:Monograph:Work

The primary contribution (bf:contribution/bflc:PrimaryContribution) was entered as a manual URI + label (using "Enter your own URI and label) since it was a newly created authority record and not yet in LC's database or in id.loc.gov or in the QA lookup. It received this error message:

![image](https://user-images.githubusercontent.com/5078198/150023822-97d1f850-ecb7-4dbe-b39f-9df49a4f6672.png)

This should convert to MARC, though admittedly the label will be imperfect if a date is involved. But this one should look like this:

100 1# $a Storopoli, Jose $e Author $0 http://id.loc.gov/authorities/names/no2022005382

**Comment by NancyL** [comment]
I will add that the template has been changed slightly since this was first reported and the name & URI had to be re-entered. The problem, however, is unchanged. It still reports an error and prevents conversion.

**Comment by justinlittman** [comment]
Yes, the code does not handle the case when there is a resolvable URI (e.g., from id.loc.gov), but that URI does not yet resolve (results in 404).
